pages/common/pnmindex.md

```diff
@@ -0,0 +1,21 @@
+# pnmindex
+
+> Build a visual index of multiple PNM images.
+> See also: `pamundice`.
+> More information: <https://netpbm.sourceforge.net/doc/pnmindex.html>.
+
+- Produce an image containing thumbnails of the specified images in a grid:
+
+`pnmindex {{path/to/input1.pnm path/to/input2.pnm ...}} > {{path/to/output.pnm}}`
+
+- Specify the size of the (quadratic) thumbnails:
```

Comment:
```suggestion
- Specify the size of the quadratic thumbnails:
```
Do we need to enclose it inside brackets?

Reply:
I wanted to express that this does not only set the sizes of the quadratic thumbnail, but that all thumbnails are quadratic.